src/protos/chunk_server.proto

```diff
@@ -5,21 +5,28 @@ package protos;
 message ChunkServerLocation {
   // The hostname of this chunk server location
   string server_hostname = 1;
+
   // The port that this chunk server is listening on for requests
   uint32 server_port = 2;
 }
 
 message ChunkServer {
+  // Used as the unique identifier for this chunkserver.
   ChunkServerLocation location = 1;
-  // Amount of memory in megabytes available on the server
-  uint32 available_memory_mb = 2;
+
+  // Amount of disk in megabytes available on the server
+  uint32 available_disk_mb = 2;
+
+  // The chunk handles for chunks on this server.
+  repeated string stored_chunk_handles = 3;
```

Comment:
Yes, and that is why we have the FileChunk below. But only want the chunkserver stored in the chunkservermgr to  store just chunkhandles and not their version too. The metadata manager is the source of versions. We only need to compare version when chunkserver reports some chunks to the mgr, and we will compare it with the version that the metadatamgr has. This will let us know if the chunkserver has a stale version and ask it to delete it. Don't want to be managing versions in two places. So version is only reported in the ReportChunkServerRequest, but will make the way the versions are stored there better, once I start making the chunkserver call. Will add a todo in  ReportChunkServerRequest.

For the clients read/write, according to the paper, master doesn't know which version is on each chunkserver, it only knows where the chunk is stored. And that's why we compare the version reported by the chunkserver with the version in the metadata mgr when the chunkserver reports itself. And ask the chunkserver to delete the chunks with stale version.